packages/peregrine/lib/Apollo/deleteCacheEntry.js

```diff
@@ -21,14 +26,48 @@ export const deleteCacheEntry = async (client, predicate) => {
         return;
     }
 
+    // Remove from the active cache.
     Object.keys(client.cache.data.data).forEach(key => {
         if (predicate(key)) {
             client.cache.data.delete(key);
         }
     });
 
-    // Immediately persist the cache changes.
+    // Immediately persist the cache changes to the active cache storage.
     if (client.persistor) {
         await client.persistor.persist();
     }
 };
+
+const deleteInactiveCachesEntry = async (client, predicate) => {
+    const activeApolloCacheLocalStorageKey =
+        client.persistor.persistor.storage.key;
+
+    const isAnInactiveApolloCache = ([key]) => {
+        // Keep this in sync with the cache names used by your storefront.
+        return (
+            key.startsWith('apollo-cache-persist') &&
```

Comment:
I do think this deserves to be an export of a constants file like `peregrine/lib/Apollo/constants.js`. Then we could import here and in the main app where we define the cache key.

Reply:
Done: https://github.com/magento/pwa-studio/pull/2992/commits/0d8cc98dc0a773c55b1114585e4e110ab89f83d4